client/js/table-model.js

```diff
@@ -16,6 +17,27 @@ class TableModel {
   _setValue(location, value) {
     this.data[this._getCellId(location)] = value;
   }
+
+  findSum(column) {
+    let sum = 0;
+    for (let location in this.data) {
+      let value = parseInt(this.data[location], 10);
+      if (location.indexOf(`${column}:`) !== -1 &&
```

Comment:
This is a fine implementation, BUT can you think of a way to use .filter() to produce the same filtering effect?